components/LessonCard.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+
+import '../scss/lessonCard.scss'
+
+type Props = {
+  coverImg: string
+  title: string
+  challengeCount: number
+  lessonCount: number
+  hourCount: number
+  description: string
+}
+
+const LessonCard: React.FC<Props> = props => {
+  return (
+    <>
+      <div className="card shadow-sm mt-5">
+        <div className="card-body p-1">
```

Comment:
Could you removed the class `p-1` -> `<div className="card-body">`
I removed the card-body padding `padding: 1.25rem;`
<img width="319" alt="Screen Shot 2020-02-27 at 9 20 17 AM" src="https://user-images.githubusercontent.com/25457563/75470108-ff0e5900-5944-11ea-8111-c7c974044fb9.png">

Reply:
Removing p-1 adds more padding than needed since cards are almost to edge